rust/bitcoin_wallet/Makefile

```diff
@@ -0,0 +1,13 @@
+.PHONY: all
+all: build
+
+.PHONY: build
+.SILENT: build
+build:
+	cargo build --target wasm32-unknown-unknown --package bitcoin_wallet --release
+	ic-cdk-optimizer target/wasm32-unknown-unknown/release/bitcoin_wallet.wasm -o target/wasm32-unknown-unknown/release/bitcoin_wallet_opt.wasm
```

Comment:
`ic-cdk-optimizer` is not needed with dfx 0.11.0 anymore if you build this with `dfx build`. Alternatively, the optimizer has to be mentioned in the prerequisites